src/clr-angular/utils/drag-and-drop/if-dragged.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {Directive, OnDestroy, TemplateRef, ViewContainerRef} from "@angular/core";
+import {Subscription} from "rxjs/Subscription";
+
+import {ClrDragEvent} from "./interfaces/drag-event";
+import {ClrDragEventListener} from "./providers/drag-event-listener";
+
+@Directive({selector: "[clrIfDragged]"})
```

Comment:
So from our call you mentioned that this directive will be placed on the ghost element. So thats why we render it only after `dragStart`. May be add a comment here to describe what this directive does.

Reply:
Sure, will do that.

Reply:
@youdz suggested this for all popover implementations. You should check with him once before we implement something similar here with a new InjectionToken.

Reply:
I don't see why there would be an InjectionToken here, the current structure/injection is fine.

Reply:
I commented on the wrong thread. Please check the conversation below.